client-angular/src/app/app.component.ts

```diff
@@ -0,0 +1,26 @@
+import { Component, CUSTOM_ELEMENTS_SCHEMA } from '@angular/core';
+import { RouterOutlet } from '@angular/router';
+import { Router } from '@angular/router';
+import { configureFoundationLogin } from './share/foundation-login';
+
+// Genesis Components
+import './share/genesis-components';
+
+
+@Component({
+  selector: 'app-root',
```

Comment:
You can use `{{rootElement}}` here and in the template to match standard seed.

Reply:
done by @ArturKrasinski

Reply:
correct, used ```{{rootElement}}```